src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -129,6 +129,11 @@ internal static string ReturnCommonPathRootFromTargetSpecifiersIfOneExists(IEnum
 
         public override int Run(AnalyzeOptions analyzeOptions)
         {
+            if (analyzeOptions.TargetFileSpecifiers == null || !analyzeOptions.TargetFileSpecifiers.Any())
```

Comment:
>analyzeOptions.TargetFileSpecifiers == null || !analyzeOptions.TargetFileSpecifiers.Any()) [](http://example.com/codeflow?start=16&length=90)

You can write this line of code more concisely as:

if (!analyzeOptions.TargetFileSpecifiers?.Any() == true)

Reply:
please merge as Michael's input.
Maye need to move `!` since can not use `!` on `null`
`if (analyzeOptions.TargetFileSpecifiers?.Any() != true)`